### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by mpa-mxiang**
For this PR, I:

- Build interactive console apps.
- Run a program using the command line.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  def execute_option(option)
+    case option
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_teacher
+    when 4
+      create_student
```
Comment (Olamarx) [inline app.rb]: - [x] Both create teacher and create student are under create person and not separated, the fact that you separated it makes no space for list rentals but only create rentals. I will implore you to add it.

### Expected UI
image

### Your UI
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @mpa-mxiang,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Readme professional ✔️
- Readable codebase ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @mpa-mxiang, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good console UI design :white_check_mark: 
- [x] Application works as expected :white_check_mark: 
- [x] Pull request has a descriptive title and summary :white_check_mark:   
- [x] Implemented all the required changes from the previous reviewer :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mpa-mxiang** [comment]
Thank you @Felix45 for the approval!

**Pull request merged**